<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchChangeDialog</class>
 <widget class="QDialog" name="BatchChangeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch-Änderung</string>
  </property>
  <widget class="QGroupBox" name="grp_Fields">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>281</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Datenfelder</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>261</width>
      <height>203</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QCheckBox" name="chb_Type">
       <property name="text">
        <string>Typ</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbb_Type"/>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="chb_Category">
       <property name="text">
        <string>Kategorie</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbb_Category"/>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="chb_SubCategory">
       <property name="text">
        <string>Unterkategorie</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cbb_SubCategory"/>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="chb_Format">
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cbb_Format"/>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="chb_Storage">
       <property name="text">
        <string>Lagerort</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="cbb_Storage"/>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="chb_StorageDetails">
       <property name="text">
        <string>Details</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btn_BatchUpdate">
      <property name="text">
       <string>Ändern</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_Cancel">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
